build.gradle

```diff
@@ -28,6 +28,9 @@ repositories {
 subprojects {
     repositories {
         mavenCentral()
+        maven {
+            url "https://dl.bintray.com/kotlin/kotlinx.html"
```

Comment:
What version of kotlinx.html do we need? Maven cental has 0.7.3 only. Maybe I could ping the authors of kotlinx.html to deploy old version to Maven Central

Reply:
the kotlinx-html-jvm-0.7.1.pom is missed but required for org.jetbrains.lets-plot:lets-plot-batik:2.0.2. That would be great if you can ping them about some release as I also would prefer not to add these three lines 😄

Reply:
let me know when it should/can be changed

Reply:
yeah, what's the problem, we could merge it as is and solve the dependency issue in the background (I have such experience with a few libraries, sometimes ping to authors is just enough)